test/cli/printer.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Printer = require('../../cli/printer.js');
+const assert = require('assert');
+const fs = require('fs');
+const sampleResults = require('../results/sample.json');
+
+/* global describe, it, beforeEach */
+
+describe('Printer', () => {
+  beforeEach(() => {
+    this.printer = new Printer();
+  });
+
+  it('has valid output modes', () => {
```

Comment:
this should be covered by the following tests. Better not to assert things about the internal state